glommio/src/reactor.rs

```diff
@@ -45,7 +45,6 @@ use crate::{
     },
     IoRequirements,
     IoStats,
-    Latency,
```

Comment:
I don't think your heuristic (as described in the commit message) makes sense.
Why send "other tasks" on the latency ring? Other tasks will include stuff like grabbing the file size from metadata (statx). If you don't think reading a file should be in the latency ring, why do you think stat should be?

I am sympathetic towards moving stuff out of the latency ring (although I think there is a case to be made for rx networking), but then stuff that goes into it should be explicitly marked, and not "the rest".

Reply:
That makes sense, I changed the behavior to what you suggested. I also added `connect/accept` to the latency ring as well, as those are also important for network operations.